.travis.yml

```diff
@@ -62,7 +62,7 @@ install:
 script:
   - mkdir build
   - cd build
-  - cmake -DCMAKE_CXX_COMPILER=$COMPILER ..
+  - EIGEN3_INCLUDE_DIR=eigen cmake -DCMAKE_CXX_COMPILER=$COMPILER ..
```

Comment:
I think this line should read:
`cmake -DCMAKE_CXX_COMPILER=$COMPILER -DEIGEN3_INCLUDE_DIR=eigen`